stories/ExampleSlider.jsx

```diff
@@ -241,4 +243,90 @@ storiesOf('Slider', module)
   ))
   .add('Disabled', () => (
     <LabeledSlider disabled />
-  ));
+  ))
+  .add('RTL', () => (
+    <DirectionProvider direction={DIRECTIONS.RTL}>
+      <LabeledSlider />
+    </DirectionProvider>
+  ))
+  .add('RTL Multiple Handle', () => (
+    <DirectionProvider direction={DIRECTIONS.RTL}>
+      <LabeledSlider values={[0, 100]} />
+    </DirectionProvider>
+  ))
+  .add('RTL Pits', () => {
+    function PitComponent({ style, children }) {
+      return (
+        <div
+          style={{
+            ...style,
+            background: '#a2a2a2',
+            width: 1,
+            height: children % 10 === 0 ? 12 : 8,
+            top: 20,
+          }}
+        />
+      );
+    }
+    PitComponent.propTypes = {
+      style: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+      children: PropTypes.number,
+    };
+    PitComponent.defaultProps = {
+      style: null,
+      children: null,
+    };
+
+    return (
+      <DirectionProvider direction={DIRECTIONS.RTL}>
+        <LabeledSlider
+          pitComponent={PitComponent}
+          pitPoints={[0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55, 60, 65, 70, 75, 80, 85, 90, 95, 100]} // eslint-disable-line max-len
+          snap
+          snapPoints={[0, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100]}
+          values={[40, 80]}
+        />
+      </DirectionProvider>
+    );
+  })
+  .add('RTL Custom Handle', () => {
+    function MyHandle({
+      styles, css, style, handleRef, ...passProps
```

Comment:
all these object keys should be in their own line, since the entire literal doesn’t fit on its own line.